src/wasm/mod.rs

```diff
@@ -334,6 +334,36 @@ impl PairedBitBox {
         Ok(psbt.to_string())
     }
 
+    #[wasm_bindgen(js_name = btcSignMessage)]
+    pub async fn btc_sign_message(
+        &self,
+        coin: types::TsBtcCoin,
+        script_config: Option<types::TsBtcScriptConfigWithKeypath>,
```

Comment:
pls make this not an `Option<>`, it is requried after all. I missed this in the previous review - you'll also need to change it in btc.rs.

Reply:
Okay, changing it in the Rust side too.